code/modules/clothing/suits/jobs.dm

```diff
@@ -141,28 +141,34 @@ obj/item/clothing/suit/apron/overalls/blue
 	icon_state = "hos_jacket"
 	item_state = "hos_jacket"
 
-//Detective
-
-/obj/item/clothing/suit/storage/det_jacket
-	name = "detective's jacket"
-	desc = "Stylish yet comfortable professional jacket manufactured by CL corporation for detectives."
+/obj/item/clothing/suit/storage/security/investigator
+	name = "investigator jacket"
+	desc = "An investigator jacket. Stylish, professional, yet comfortable."
```

Comment:
try not to put changes into bugfixes, thanks

Reply:
not really a change if the jacket has been used for the investigators before.